pogobuf/lehmer.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
This is a Lehmer RNG apparently. Is there an npm module for this? I'd prefer that over adding an extra file.